Juice/Classes/ChargeScaleDisplay.swift

```diff
@@ -18,9 +19,9 @@ protocol ChargeScaleDisplay {
 }
 
 enum SerializableChargeScaleDisplayConstants {
-    static let title = "title"
-    static let detents = "detents"
-    static let defaultDetentString = "default"
+    static let title = NSLocalizedString("title", comment: "title")
```

Comment:
These should not be localized as they are part of the plist serialization format.